internals/daemon/api_exec.go

```diff
@@ -64,7 +64,20 @@ func v1PostExec(c *Command, req *http.Request, _ *userState) Response {
 	// Check up-front that the executable exists.
 	_, err := exec.LookPath(payload.Command[0])
```

Comment:
I thought at first that perhaps the executable lookup could happen relative to the working directory, and therefore I was wondering if we should check the working directory first. It seems that ```LookPath()``` will only consult $PATH or an absolute path for verification, so that does not seem like a valid use case.